### DISCUSSION TITLE
❓ [Question] Can you convert model that operates on custom classes?

### DISCUSSION THREAD

**Discussion by MarekPokropinski** [pytorch/TensorRT#1073] [Q&A]
## ❓ Question

I have a torch module that creates objects of custom classes that have tensors as fields. It can be torch.jit.scripted but torch.jit.trace can be problematic. When I torch.jit.script module and then torch_tensorrt.compile it I get the following error: `Unable to get schema for Node %317 : __torch__.src.MyClass = prim::CreateObject() (conversion.VerifyCoverterSupportForBlock)`

## What you have already tried

torch.jit.trace avoids the problem but introduces problems with loops in module.

## Environment

> Build information about Torch-TensorRT can be found by turning on debug messages

 - PyTorch Version (e.g., 1.0): 1.10.2
 - CPU Architecture: intel
 - OS (e.g., Linux): linux
 - How you installed PyTorch (`conda`, `pip`, `libtorch`, source): pip
 - Build command you used (if compiling from source):
 - Are you using local sources or building from archives: from archives
 - Python version: 3.8
 - CUDA version: 11.3
 - GPU models and configuration:  rtx 3090
 - Any other relevant information:

**Comment by narendasan** [comment]
You should be able to use the method in this issue to compile your model https://github.com/NVIDIA/Torch-TensorRT/issues/876

**Comment by narendasan** [comment]
Also make sure your operation is registered through torchbind https://pytorch.org/tutorials/advanced/torch_script_custom_ops.html
